src/hooks/useProposal/useProposal.ts

```diff
@@ -268,6 +268,22 @@ export function useProposal(proposal: ProposalTypes.Struct) {
     [client, queryClient],
   );
 
+  const addNewAccounts = useCallback(
+    async (currencies: string[]) => {
+      try {
+        await client.account.request({
+          currencyIds: currencies,
+        });
+        // TODO Maybe we should also select the requested account
+      } catch (error) {
+        console.error("request account canceled by user");
+      }
```

Comment:
This is not the only error possible we should show a toast here unless it's cancelled by the user by check the error